doc/JSON-RPC-interface.md

```diff
@@ -5,6 +5,36 @@ The headless daemon `bitcoind` has the JSON-RPC API enabled by default, the GUI
 option. In the GUI it is possible to execute RPC methods in the Debug Console
 Dialog.
 
+## Endpoints
+
+There are two JSON-RPC endpoints on the server:
+
+1. `/`
+2. `/wallet/<walletname>/`
+
+### `/` endpoint
+
+This endpoint is always active and can service both wallet and non-wallet
+requests when there are zero or one wallets loaded.
+
+### `/wallet/<walletname>/` endpoint
+
+This endpoint is only activated when the wallet component has been compiled in
```

Comment:
It is also always active for non-wallet requests before any wallets are loaded.

Reply:
(but only when the wallet is compiled in).

Also I think i've covered that it handles both types of request when no wallet is loaded in the Quirks section?